### ISSUE TITLE
Super Hang on on Analogue Pocket

### ISSUE THREAD

**Issue by Wolfyteemo** [jotego/jtcores#902]
Super Hangon was working until latest update. Now when a race starts the bike will speed up on it's own, and also after entering any direction the bike will pull constantly to the left. Tried resetting, tried reinstalling it. Tried on a fresh SD card. Still has same problem, but the game was working perfectly before the game update.

**Comment by Dawbo66** [comment]
Is this docked or both ?, there's an issue docked with some controllers pulling to the left, I would put an older version on until it's fixed.

**Comment by Wolfyteemo** [comment]
Not just pull to the left, it accelerates as well. This is not docked.

**Comment by Dawbo66** [comment]
Yes your right also it starts and the select track screen and not the Super Hang On game screen.

**Comment by jotego** [comment]
This might be related to a change done to the Pocket framework in order to have analog controls on JTWC. We have to review it.

**Comment by mattyv316** [comment]
Having this problem as well, even on the latest. Going back to d29c069 (11/29) fixes the issue. Looks like the next 3b2a2fa (12/14) build is where I started having the issue.

**Comment by jotego** [comment]
The commits referred above are unrelated to the control system.
This bug was fixed a while ago, probably in this [commit](https://github.com/jotego/pocket/commit/f831bb29eadd00068010da59468ed04445c19dbc) of the Pocket submodule.
